app/src/main/java/com/framgia/wsm/data/source/remote/api/service/FireBaseInstanceIDService.java

```diff
@@ -0,0 +1,23 @@
+package com.framgia.wsm.data.source.remote.api.service;
+
+import android.util.Log;
+import com.google.firebase.iid.FirebaseInstanceId;
+import com.google.firebase.iid.FirebaseInstanceIdService;
+
+/**
+ * Created by ASUS on 14/06/2017.
+ */
+
+public class FireBaseInstanceIDService extends FirebaseInstanceIdService {
+    private static final String TAG = "FireBaseInstanceIDService";
+
+    @Override
+    public void onTokenRefresh() {
+        getToken();
+        Log.d(TAG, "Refreshed token: " + getToken());
```

Comment:
ERROR: [LongLogTag] The logging tag can be at most 23 characters, was 25 (FireBaseInstanceIDService)
 Log tags are only allowed to be at most 23 tag characters long.